includes/REST/WithdrawController.php

```diff
@@ -579,16 +579,20 @@ public function batch_items( $request ) {
      * @return data
      */
     public function prepare_item_for_response( $withdraw, $request ) {
-        $methods = dokan_withdraw_get_methods();
+        $methods     = dokan_withdraw_get_methods();
+        $vendor      = $this->get_user_data( $withdraw->get_user_id() );
+        $get_details = [ $withdraw->get_method() => ! empty( $withdraw->get_details() ) ? maybe_unserialize( $withdraw->get_details() ) : array() ];
```

Comment:
why we are using [] ? this line of code looks alien to me, can you please explain this line a bit?